assets/src/domain/eventEditor/ui/tickets/ticketsList/filterBar/controls/TicketsChainedButton.tsx

```diff
@@ -15,18 +15,19 @@ interface TicketsChainedButtonProps {
  * will appear, otherwise ALL tickets will appear (subject to other filters)
  */
 const TicketsChainedButton: React.FC<TicketsChainedButtonProps> = ({ isChained, toggleIsChained }) => {
-	const className = classNames('ee-filter-bar__btn', {
+	const className = classNames('ee-filter-bar__btn ee-filter-bar__chain', {
 		'ee-filter-bar__btn--active': isChained,
+		'ee-filter-bar__chain--active': isChained,
 	});
 
 	const icon = isChained ? Icon.LINK : Icon.UNLINK;
+	const label = __('link');
 	const tooltip = isChained
 		? __('tickets list is linked to dates list and is showing tickets for above dates only')
 		: __('tickets list is unlinked and is showing tickets for all event dates');
 
 	return (
-		<>
-			<label className='esprs-button-label screen-reader-text'>{__('Only Show Tickets for Dates Above')}</label>
+		<BaseInput label={label}>
```

Comment:
i thought of the perfect name for this... convert it to an HOC and call it `withLabel` and then wrap the input exports at the source (or the imports in the implementation)

Reply:
I've added `withLabel` HOC : https://github.com/eventespresso/event-espresso-core/pull/2602/commits/a6fa54c21355a004cb90cf4ceb3f6f17b9797f02

Reply:
since `withLabel` needs `label` prop which is not passed to `EspressoButton`, we can't just `wrap the exports at the source`

Reply:
TBH this approach with HOC is a bit convoluted.
A simpler solution I'd see is to just pass `label` to `EspressoButton` and deal at that level or even simpler, create a component `EspressoButtonWithLabel` which is not a HOC but a wrapper around `EspressoButton` which accepts `label` along those `EspressoButtonProps`